packages/cli/mocks/mock-project-with-root-imports/zos.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "mock-project-with-root-imports",
+  "zosversion": "2.2",
```

Comment:
We should remember to migrate these `zosversion` references before merging this commit into master, as https://github.com/zeppelinos/zos/pull/1000 has already been merged.

Reply:
Good catch!